src/test/scenarios/compute/output/core_Incremental/src/azure-cli/azure/cli/command_modules/vm/tests/latest/example_steps.py

```diff
@@ -22,3 +22,13 @@ def step_virtual_machine_assess_patch(test, rg, checks=None):
              '--vm-name "myVMName"',
              checks=checks)
 
+
+@try_manual
+def step_virtual_machine_assess_patch_min(test, rg, checks=None):
+    if checks is None:
+        checks = []
+    test.cmd('az vm virtual-machine assess-patch '
+             '--resource-group "{rg}" '
+             '--vm-name "myVMName"',
+             checks=checks)
```

Comment:
Do we still need a minimal tests if the generated tests already are minimal.

Reply:
created a bridge call in latest commit